web/admin/src/editCell.js

```diff
@@ -78,12 +78,24 @@ export class CellEdit extends Component {
      }
 
      confirmEdit = () => {
-        var sessionName = this.state.value.trim()
-        if (sessionName) {
-            this.props.confirmEdit(this.props.task, this.state.value)
-            this.setState({action: "state", value : this.props.task.sessionName});
-            this.myInp.blur()
+        var named = this.state.value.trim()
```

Comment:
`const` and `let` give better guarantees against developer mistakes than `var`.  `named` doesn't change, so it can be `const`.  `status` can change, so it can be `let`